v2.2/follower-reads.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Follower Reads
+summary: To reduce latency for `SELECT` operations, you can choose to have the closest node serve the request using the "Follower Reads" feature.
```

Comment:
something of a nit: Maybe you should just say read queries? It can also speed up backups